<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-E1E9EHH" ignored="0" name="Test" tests="4" failures="2" timestamp="07 Dec 2021 04:41:19 GMT" time="29.659" errors="0">
  <testcase name="createBooking" time="9.553" classname="datashare.TestNG_DataStore_ThreadLocal_01"/>
  <testcase name="createBooking" time="9.556" classname="datashare.TestNG_DataStore_ThreadLocal_02"/>
  <testcase name="retrieveBooking" time="0.006" classname="datashare.TestNG_DataStore_ThreadLocal_01">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at datashare.TestNG_DataStore_ThreadLocal_01.retrieveBooking(TestNG_DataStore_ThreadLocal_01.java:51)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- retrieveBooking -->
  <testcase name="retrieveBooking" time="20.012" classname="datashare.TestNG_DataStore_ThreadLocal_02">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at datashare.TestNG_DataStore_ThreadLocal_02.retrieveBooking(TestNG_DataStore_ThreadLocal_02.java:55)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- retrieveBooking -->
</testsuite> <!-- Test -->
